### DISCUSSION TITLE
add light distance and light angle focus

### DISCUSSION THREAD

**Discussion by adhhrg** [leomoon-studios/blender-light-studio#34] [Ideas]
![gvsvc9A](https://user-images.githubusercontent.com/63681591/103426937-2e0be180-4b8b-11eb-956c-90de7659f205.png)
light distance

https://user-images.githubusercontent.com/63681591/103427010-ce620600-4b8b-11eb-8b29-20f162c82d1f.mp4

light angle

i think these will make the addon a lot better

**Comment by leomoon** [comment]
The main reason for adding area light to LightStudio was to support EEVEE.

Are both of these features supported in EEVEE?
